src/__tests__/Link.tsx

```diff
@@ -43,6 +43,6 @@ describe('Link', () => {
   })
 
   it('respectes the "color" prop when "muted" prop is also passed', () => {
-    expect(render(<Link muted color="black" />)).toMatchSnapshot()
+    expect(render(<Link muted color="text.inverse" />)).toMatchSnapshot()
```

Comment:
tried to pass a functional color here that I think would never coincide with the "muted" color in any given theme.